spark-plugins/src/main/java/co/cask/hydrator/plugin/batch/spark/NaiveBayesClassifier.java

```diff
@@ -83,11 +83,6 @@ public Config(String fileSetName, String path, String fieldToClassify, String fi
     }
   }
 
-  // for unit tests, otherwise config is injected by plugin framework.
-  public NaiveBayesClassifier(Config config) {
```

Comment:
for future reference, its ok to have this too, plugin framework calls this constructor if it exists.